g11n-ws/vip-manager-l10n/src/main/java/com/vmware/l10n/utils/S3Util.java

```diff
@@ -33,7 +35,7 @@
 	private static Random random = new Random(System.currentTimeMillis());
```

Comment:
Delete this line directly?

Reply:
it need to write a random to lock file

Reply:
Because version id is used instead of file content, this isn't necessary any more.

Reply:
S3 write object API need content is not null

Reply:
How about use a constant instead of random?